packages/lab/src/Flow/Flow.spec.ts

```diff
@@ -117,19 +118,22 @@ test.describe("Node", () => {
     const node = await selectNode("ML Model", "ML Model Prediction", true);
     await dragToPosition1(node);
 
-    const createdNode = flowCanvasLocator.getByRole("button", {
-      name: "ML Model Prediction",
-    });
+    const createdNode = flowCanvasLocator
+      .getByRole("button", {
+        name: "ML Model Prediction",
+      })
+      .first();
     expect(await createdNode.all()).toHaveLength(1);
     await flowCanvasLocator
       .getByRole("button", { name: "ML Model Prediction" })
+      .first()
       .hover();
     await page.getByRole("button", { name: "Duplicate" }).hover();
     await expect(
       page.getByRole("tooltip", { name: "Duplicate" }),
     ).toBeVisible();
     await page.getByRole("button", { name: "Duplicate" }).click();
-    expect(await createdNode.all()).toHaveLength(2);
+    expect(await createdNode.all()).toHaveLength(1);
```

Comment:
The length should still be 2 here since we are testing if the node gets duplicated. If we need to update this to 1 then there's probably something that needs fixing.